Makefile

```diff
@@ -78,11 +94,144 @@ remove-firefox-id:
 beta-firefox-zip: remove-firefox-id
 	cd build/firefox/release/ && web-ext build
 
+###--- Build dir preparation ---###
+## artifacts for integration-tests
+setup-artifacts-dir:
+	rm -rf integration-test/artifacts
+	mkdir -p integration-test/artifacts/screenshots
+	mkdir -p integration-test/artifacts/api_schemas
+
+# create build dir ready for source
+ifeq ('$(browser)','chrome-mv3')
+$(BUILD_DIR)/public/js: shared/data/bundled/smarter-encryption-rules.json
+else
+$(BUILD_DIR)/public/js:
+endif
+	mkdir -p $(BUILD_DIR)/public/js/
+	mkdir -p $(BUILD_FOLDERS)
+
+# fetch SE data for bundled SE rules
 shared/data/smarter_encryption.txt:
 	curl https://staticcdn.duckduckgo.com/https/smarter_encryption.txt.gz | gunzip -c > shared/data/smarter_encryption.txt
 
+# build SE rules for MV3
 shared/data/bundled/smarter-encryption-rules.json: shared/data/smarter_encryption.txt
 	npm run bundle-se
 
+# fetch integration test data
+integration-test/artifacts/attribution.json: node_modules/privacy-test-pages/adClickFlow/shared/testCases.json setup-artifacts-dir
+	mkdir -p integration-test/artifacts
+	cp $< $@
```

Comment:
@jonathanKingston this replaces `attributionTestCases.mjs`, and moves to a local copy rather than fetching from glitch.